self-reflection-1-skygennette.qmd

```diff
@@ -0,0 +1,63 @@
+---
+
+## About you
+
+1.  Please tell me a little bit about your research interests and how you imagine integrating spatial data in your research.
+
+I'm interested in using remote sensing and artificial intelligence as tools to improve ecological monitoring. I plant on integrating spatially explicit spectral data derived from remote sensing platforms including satellites and Unoccupied Aerial Systems. 
+
+2.  How do you learn best, and what do you feel is your biggest strength as a student?
+
+Especially in the context of this type of course, I learn best by walking through code and annotating during class. I feel that my biggest strength is my ability to work and problem-solve independently.
+
+
+3.  If you want to, you can set a goal for this semester for something you want to work on in the context of this course (outside of the topic matter itself) -- for example, using StackOverflow to get answers to analysis questions or using git to track progress on homeworks. If you choose to set this type of goal, what can you do to help support yourself in working toward that goal? How can I help you to meet this goal?
+
+I want to try to make my code from this and other courses easier for myself to reference back to in the future when trying to find specific pieces of code. This might be creating separate cheatsheet documents or just a good repository in github with readme files. If you know of good systems for this, that would be helpful.
+
+4.  You can, if you choose, also let me know if there is anything else you have going on this semester that you would like me to know about (e.g., field work, other courses, prepping your qualifying exam, working outside of assistantship).
+
+I do have fieldwork that I don't anticipate will interfere with this course but may due to weather delays. 
+
+## This Course
+
+5.  Please tell me a little about your prior experience with R ("I use it all the time", "I am still learning", "I've never used it before"). What do you like most about working with R? What are the biggest obstacles for you when working with R?
+
+I have used R since I was an undergrad, and I now use it all the time. I am comfortable coding in R and like the community and resources it comes with. My biggest obstacle is that the packages I have worked with are somewhat limited, so I don't have a great idea of everything that is possible in R. 
+
+6.  Describe your experience working with spatial data and GIS (either in R, or ArcGIS, or QGIS etc.). How do you tend to use spatial data (e.g., making maps, desiging field sampling, buidling stastical models)?
+
+I have worked a lot with many forms of spatial data in R, ArcGIS, and QGIS. I have mostly worked with wildlife movement data, landcover datasets, and remotely sensed spectral data produce maps, create management plans, design field sampling, and do statistical analyses. The part I have the least amount of experience with is building statistically models with spatial data. 
```

Comment:
It sounds like you've already built the foundation of skills that I'll be teaching in the first few weeks!